hyper/http20/connection.py

```diff
@@ -482,6 +497,9 @@ def _send_cb(self, frame, tolerate_peer_gone=False):
 
         data = frame.serialize()
 
+        if frame.body_len > FRAME_MAX_LEN:  # pragma: no cover
```

Comment:
Sorry about having to make this note. I've had lots of trouble in the past keeping track of which setting applies to which direction in the communication, so I'm not entirely surprised that we've had to go back and forth on this a few times.